handlers.js

```diff
@@ -155,7 +188,11 @@ async function handle_message(reply, content) {
     handler = handlers[content.split(" ")[0]];
     if (handler != undefined) {
         try{
-            let r = await handler(content.split(" ").slice(1, 9));
+            var args = [content.split(" ").slice(1, 9)];
```

Comment:
Why not leave out the extra enclosing `[]` and refer to it differently below?

Reply:
The alternative would result in more if/else statements unless I'm missing something. Although I think even this looks cleaner.
```js
let r;
if (handler.name == reply_wait().name) {
    r = await handler(reply, content.split(" ").slice(1, 9));
}
else {
    r = await handler(content.split(" ").slice(1, 9));
}
```